hwangsae/common.h

```diff
@@ -0,0 +1,20 @@
+/**
+ *  Copyright 2019 SK Telecom Co., Ltd.
+ *    Author: Walter Lozano <walter.lozano@collabora.com>
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+#include <gio/gio.h>
```

Comment:
Nitpick: I guess `#include <glib.h>` would be enough to pull in `gchar`; no need to include GIO, which implies GObject etc., just for that.

Reply:
Noted.